cowprotocol/cowswap
# #1852/Shortening token symbols

# Summary

## Updated 2023-01-24

I really hope now I got all the places 🤞 
Checked Activity modals, pop-ups, swap and limit orders

![Screenshot 2023-01-24 at 15 54 49](https://user-images.githubusercontent.com/43217/214344071-9a12256a-187c-4654-9977-bfa518439a1c.png)
![Screenshot 2023-01-24 at 15 54 38](https://user-images.githubusercontent.com/43217/214344081-af4c37e9-06c3-4e71-b4ae-90d60057effd.png)
![Screenshot 2023-01-24 at 15 49 17](https://user-images.githubusercontent.com/43217/214344083-b6be3b2c-fa20-4f9d-8e68-9fd4f4299630.png)
![Screenshot 2023-01-24 at 15 34 17](https://user-images.githubusercontent.com/43217/214344110-d0c78819-1d96-4b3a-a1c6-e6bd3bfd86ee.png)
![Screenshot 2023-01-24 at 15 34 06](https://user-images.githubusercontent.com/43217/214344114-b00ae851-7f69-4123-8a3f-1be2610dd1ef.png)

## Testing

I've added a temporary code to make every new token added suuuuper long
Will of course revert before merging.

1. Add a token that's not in any list. For example, `0x07865c6e87b9f70255377e024ace6630c1eaa37f` on Goerli
2. Swap, limit, anything you can think of with this
* Name should be trimmed everywhere, except in titles


---

## Original

Closes #1852 

Follow up to https://github.com/cowprotocol/cowswap/pull/1843

![Screenshot 2023-01-11 at 15 34 35](https://user-images.githubusercontent.com/43217/211850622-d3b8e647-0fad-4f22-aade-283a6df28b51.png)

# To Test

1. Load token https://etherscan.io/token/0x2E2A42fbE7C7E2FfC031BAF7442dBE1F8957770a
4. Check both swap and limit orders
* The ui should not be broken

Note: further testing required in all part of the app. For that would be nice to find a georli token with a long symbol name.
Not able to test with that one on mainnet as there's no liquidity
